doc/authors.rst

```diff
@@ -1,22 +1,31 @@
-========================
+.. _authors:
+
+************************
 Authors and contributors
-========================
+************************
 
 The following people have contributed code and/or ideas to the current version
-of Elephant. The institutional affiliations are those at the time of the
+of elephant. The institutional affiliations are those at the time of the
```

Comment:
Quite honestly, I didn't remember, but yes, this would be also more consistent with "Neo". Will change throughout.